docs/tutorials/qpe_with_sampler.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>I know they can go to the textbook to learn more, but would it be possible to include at least one line about WHY it is a central building block for many quantum algorithms and/or one of the most important subroutines? We basically just say that it is important twice (1st and 2nd sentence) without giving any solid context</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I totally agree with you. This part is directly copied from the textbook as a placeholder. I will take note to explain why QPE is a central building block and why it is important.</p>